setup.cfg

```diff
@@ -86,6 +86,7 @@ install_requires =
     tabulate == 0.8.2
     testinfra == 1.19.0
     tree-format == 0.1.2
+    yamllint >= 1.15.0, < 1.16
```

Comment:
So it looks like, before this, it was an optional dependency. Do we want to make it mandatory at all times? This will mean that if users opt out of using yamllint they'll get an unnecessary dependency (with transitive deps) installed for no reason.

Also, if we're adding it here we'll need to remove entries from `tox.ini`.

Reply:
Right. As it is the only one available right now (see https://molecule.readthedocs.io/en/stable/configuration.html#lint), I'm inclined to put it there as default.

Reply:
Fair enough.